lobby.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Lobby</title>
+		<link href="https://fonts.googleapis.com/css?family=Inconsolata:700" rel="stylesheet">
+		<link href='http://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
+		<link href="https://fonts.googleapis.com/css?family=Hammersmith+One" rel="stylesheet" type="text/css" />
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<link rel="stylesheet" type="text/css" href="css/basic.css">
+		<link rel="stylesheet" type="text/css" href="css/image-icons.css">
+	</head>
+
+	<body>
+		<header class="header">
+			<nav class="navbar">
+				<a class="navbar_brand" href="#">
+					<span class="text_red">Mono</span>Banker
+				</a>
+			</nav>
+		</header>
+		<div class="content container">
+			<div class="lobby_section">
+				<div class="lobby_section__header">
+					<div class="lobby_section__title">
+						Your game
+					</div>
+					<div class="row lobby_section__gameid">
+						<input type="text" name="gameid" value="cOJzeV" size="6" readonly>
+						<div class="clipboard">
+							<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
+								<path d="M0 0h24v24H0z" fill="none"/>
+								<path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z" fill="fff"/>
+							</svg>
+						</div>
+					</div>
+				</div>
+				<div class="lobby_section__rect">
+					<div class="row">
+						<div class="lobby_section__player col-3">
+							<img src="images/users/kevin.jpg" class="avatar">
```

Comment:
Images should have `alt` attribute with text to describe image for people who cannot see it. In your case you could set it to, for instance, "Kevin photo" (or avatar).